doc/diagrams/classes/README.md

```diff
@@ -0,0 +1,14 @@
+Diagrams created with pyreverse:
+
+```
+pyreverse -o png -p pyoracc python/pyoracc/
```

Comment:
Include invocation of these pyreverse commands in the Jenkins build and publish the artefacts through Jenkins.